K2Bridge/JsonConverters/ResponseAggregationsConverter.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.JsonConverters
+{
+    using System;
+    using System.Collections.Generic;
+    using K2Bridge.Models.Request.Aggregations;
+    using K2Bridge.Models.Response;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// A converter able to deserialize the aggs element in an Elasticsearch query to <see cref="Aggregation"/>.
+    /// </summary>
+    internal class ResponseAggregationsConverter : JsonConverter
```

Comment:
To be aligned with other converters. Should we create a WriteOnlyJsonConverter class and inherit from it?

Reply:
I guess it would be nice, again for consistency with `ReadOnlyJsonConverter`...

Reply:
Fixed. And Moved ReadOnlyJsonConverter and WriteOnlyJsonConverter to a base folder.